src/Event.php

```diff
@@ -921,6 +939,26 @@ public function getExpression()
         return $this->expression;
     }
 
+    /**
+     * Get the 'from' configuration for the event if present.
+     *
+     * @return \DateTime|string|null
+     */
+    public function getFrom()
```

Comment:
Use union type for return, in both cases.

Reply:
Bump.

Reply:
Done